app/src/utils/transferTracking.ts

```diff
@@ -93,24 +93,19 @@ export function getTransferStatusFromParachain(
   const isBHChannelMsgDeliveredInSnowbridgeRes =
     'bridgeHubChannelDelivered' in transferResult &&
     transferResult.bridgeHubChannelDelivered?.success
-  /** Destination Event Index available */
-  const isDestEventIdxInSubscanXCMRes =
-    'destEventIndex' in transferResult && transferResult.destEventIndex.length > 0
+  /** Destination chain is Ethereum in XCM transfer*/
+  const isDestChainEThInSubscanXCMRes =
```

Comment:
I read a small article the other day about not uppercasing in var names because it makes it hard to read and I got to agree  👀  and these names are also getting out of order, I wonder if we can somehow organise the code in a way that we don't have to put all of this info in the naming but understand a lot of from from the context (e.g. `isDestEthereum` instead of `isDestChainEThInSubscanXCMRes`. Just thinking out loud, not a blocker and not something for now)

```suggestion
  const isDestChainEthInSubscanXcmRes =
```

Reply:
I also feel naming is a bit messy. 
For now, I'm committing, your typo fix and once we finish tracking parachains transfers, I will clean and improve general namings, eventually outsourcing some functions, etc...

Reply:
=> I've updated the naming. 
It makes more sense to do it now 🫠